Plugin/BlankCanvas/README.md

```diff
@@ -0,0 +1,121 @@
+# BlankCanvas - the plugin
+
+This plugin is a blank canvas for your plugin.
+
+It offers:
+
+1. A framework for an empty plugin
+2. A sample of how to use configuration based on a default config file stored in the Plugin jar file.
+3. A checkstyle build task (to check your Java code style)
+4. 
+## How to build BlankCanvas
+
+
+1. `git clone https://github.com/sibomots/BlankCanvas.git`
+2. `cd BlankCanvas`
+3. `gradlew setupDecompWorkspace --refresh-dependencies`
+4. `gradlew build`
+
+
+## Supar Fast Way to Test (Without an IDE)
+
+1. Setup a Forge Server
+2. Drop a release build of Sponge into the mods folder
+3. Drop your BlankCanvas plugin into the mods folder
+4. Start the server.
+
+ELSE, if you want to DEBUG with SpongeForge in place in  IDEA, then read
+on:
+
+## How debug/run within IDEA (SpongeForge + BlankCanvas)
+
+Follow the directions on GitHub for SpongePowered.  Especially these steps:
+
+1. `git clone --recursive https://github.com/SpongePowered/SpongeForge.git`
+2. `cd SpongeForge`
+3. `cp scripts/pre-commit .git/hooks`
+4. `gradlew setupDecompWorkspace --refresh-dependencies`
+5. `gradlew build`
+
+Now, Start InteliJ IDEA.
+
+Choose `Import Project`
+Select the `build.gradle` for `SpongeForge` 
+
+IDEA will take some time to configure.  Once it is settled:
+
+Choose: `File > Project Structure > Modules`
+
+Press `+`
+
+Choose the `build.gradle` of `BlankCanvas`
+
+Press `OK`
+
+IDEA will take some time to configure; once it is settled:
+
+We need to configure the Run/Debug sessions.  Do not use the advice on the SpongePowered site to have gradle do it.
```

Comment:
It is and the documentation I wrote is based on stale facts.  I've since adjusted my own work flow and posted a set of videos on the very topic.  Thanks for reminding me.